libs/pages/settings/src/lib/ui/page-organization-credentials/page-organization-credentials.tsx

```diff
@@ -0,0 +1,222 @@
+import { CloudProviderEnum, type ClusterCredentials, type CredentialCluster } from 'qovery-typescript-axios'
+import { useMemo } from 'react'
+import { useParams } from 'react-router-dom'
+import { match } from 'ts-pattern'
+import { useDeleteCloudProviderCredential } from '@qovery/domains/cloud-providers/feature'
+import { ClusterAvatar, ClusterCredentialsModal, CredentialsListClustersModal } from '@qovery/domains/clusters/feature'
+import { useOrganizationCredentials } from '@qovery/domains/organizations/feature'
+import { Button, Icon, Indicator, Tooltip, useModal, useModalConfirmation } from '@qovery/shared/ui'
+
+const toCloudProvider = (cloudProvider: ClusterCredentials['object_type']): CloudProviderEnum => {
+  return match(cloudProvider)
+    .with('AWS', () => CloudProviderEnum.AWS)
+    .with('AWS_ROLE', () => CloudProviderEnum.AWS)
+    .with('AZURE', () => CloudProviderEnum.AZURE)
+    .with('SCW', () => CloudProviderEnum.SCW)
+    .with('OTHER', () => CloudProviderEnum.ON_PREMISE)
+    .with('GCP', () => CloudProviderEnum.GCP)
+    .exhaustive()
+}
+
+export const PageOrganizationCredentials = () => {
+  const { openModal, closeModal } = useModal()
+  const { organizationId = '' } = useParams()
+  const { openModalConfirmation } = useModalConfirmation()
+  const { mutateAsync: deleteCloudProviderCredential } = useDeleteCloudProviderCredential()
```

Comment:
If you use `mutateAsync`, you need to have `try catch`. Otherwise, you can use `mutate`, it allows using the global error toast